tests/PHPStan/Analyser/data/bug-6866.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Bug6740;
```

Comment:
Namespace has a different bug number then the filename.

And another bug number in the 
Method name